doc/computation.rst

```diff
@@ -199,6 +199,41 @@ You can also use ``construct`` to compute a weighted rolling sum:
   To avoid this, use ``skipna=False`` as the above example.
 
 
+Coarsen large arrays
+====================
+
+``DataArray`` objects include a :py:meth:`~xarray.DataArray.coarsen` method.
+This method supports the block aggregation:
+
+.. ipython:: python
+
+  da = xr.DataArray(np.linspace(0, 364, num=364), dims='time',
```

Comment:
What do you think about including a 2D example in the docs, e.g., a 4x4 array to 2x2? I expect that is closer to the typical use case.

Reply:
Changed to 2d example